storehaus-hbase/src/main/scala/com/twitter/storehaus/asynchbase/AsyncHBaseStore.scala

```diff
@@ -36,46 +36,56 @@ trait AsyncHBaseStore {
   protected val column: String
   protected val threads: Int
   protected val client: HBaseClient
-  protected val futurePool = FuturePool(Executors.newFixedThreadPool(threads))
 
   def validateConfiguration() {
     import org.apache.commons.lang.StringUtils.isNotEmpty
 
-    require(!quorumNames.isEmpty, "Zookeeper quorums are required")
+    require(quorumNames.nonEmpty, "Zookeeper quorums are required")
     require(isNotEmpty(columnFamily), "column family is required")
     require(isNotEmpty(column), "column is required")
     require(isNotEmpty(table), "table is required")
-    client.ensureTableExists(table.as[Array[Byte]])
+    client.ensureTableExists(table)
```

Comment:
I tried awaiting the result of this async call, however it hangs forever. As is, it doesn't do anything anyway.